docs/architecture/3rd party technical review/000-technical-review-records.md

```diff
@@ -0,0 +1,40 @@
+# 3rd-Party Technical Review Format
+
+## Context
+
+We are using succefully lightweight architecture decision records to document decisions being made about architecture.
+Last year legislation changes around data privacy (GDPR, CCPA, PECR) and their impact on our obligations means we need to increase our technical measures to ensure the safety of our readers data through:
+  * Reducing number of third-party    
```

Comment:
A little inconsistent with the main title.

There's no entry in the [style guide], but **third party** can be a noun (no hyphen, can be plural) or adjective (with hyphen, no plural)

```suggestion
  * Reducing the number of third parties
  * Reducing the number of third-party scripts
```